variables.tf

```diff
@@ -1,3 +1,31 @@
+variable "vpc_id" {
+  type        = string
+  description = "The ID of the VPC."
+}
+
+variable "istio_external_gateway_lb_certs" {
+  type        = string
+  description = "The certificates for the Istio external gateway load balancer."
```

Comment:
is it an array or a comma-separated list?

Reply:
changed to list of string and joining with , on actual call